packages/components/src/tabs/test/index.tsx

```diff
@@ -1172,37 +1172,79 @@ describe( 'Tabs', () => {
 				).not.toBeInTheDocument();
 			} );
 		} );
-
-		describe( 'When `selectOnMove` is `true`', () => {
-			it( 'should automatically select a newly focused tab', async () => {
-				render( <ControlledTabs tabs={ TABS } selectedTabId="beta" /> );
-
-				await press.Tab();
+		describe( 'When `selectedId` is changed by the controlling component', () => {
+			it( 'should automatically update focus if the selected tab already had focus', async () => {
+				const { rerender } = render(
+					<ControlledTabs
+						tabs={ TABS }
+						selectedTabId="beta"
+						selectOnMove={ false }
+					/>
+				);
 
 				// Tab key should focus the currently selected tab, which is Beta.
+				await press.Tab();
 				expect( await getSelectedTab() ).toHaveTextContent( 'Beta' );
 				expect( await getSelectedTab() ).toHaveFocus();
 
-				// Arrow keys should select and move focus to the next tab.
-				await press.ArrowRight();
+				rerender(
+					<ControlledTabs
+						tabs={ TABS }
+						selectedTabId="gamma"
+						selectOnMove={ false }
+					/>
+				);
+
+				// When the selected tab is changed, it should automatically receive focus.
 				expect( await getSelectedTab() ).toHaveTextContent( 'Gamma' );
 				expect( await getSelectedTab() ).toHaveFocus();
 			} );
-			it( 'should automatically update focus when the selected tab is changed by the controlling component', async () => {
+			it( 'should not automatically update focus if the selected tab was not already focused', async () => {
 				const { rerender } = render(
-					<ControlledTabs tabs={ TABS } selectedTabId="beta" />
+					<ControlledTabs
+						tabs={ TABS }
+						selectedTabId="beta"
+						selectOnMove={ false }
+					/>
 				);
 
+				expect( await getSelectedTab() ).toHaveTextContent( 'Beta' );
+
 				// Tab key should focus the currently selected tab, which is Beta.
 				await press.Tab();
-				expect( await getSelectedTab() ).toHaveTextContent( 'Beta' );
 				expect( await getSelectedTab() ).toHaveFocus();
+				// Arrow left to focus Alpha, which is not the currently
+				// selected tab
+				await press.ArrowLeft();
```

Comment:
If we do run this test with `selectOnMove={true}`, we will likely need to find a different way to focus the Aplha tab